library/src/main/java/com/okta/appauth/android/OktaAppAuth.java

```diff
@@ -783,7 +832,10 @@ public void run() {
      * Use of this handler is optional. After a delay, this handler will warm up
      * the Custom Tabs browser used for authentication with the supplied login hint;
      * the delay avoids constantly re-initializing the browser while the user is typing.
+     * <p/>
+     * NOTE: Going to be deleted in v 1.0.0
      */
+    @Deprecated
```

Comment:
IMHO, just delete it now, we are pre 1.0 and can make breaking changes, we just need to make sure we document how to upgrade.

That said, there is also nothing wrong with marking them deprecated now, and going back through later and cleaning this up. (as that makes for smaller diffs).
If we are going to keep them for now add a `deprecated javadoc tag too pointing to the replacement)

Reply:
+1, just add javadoc tag then